### PR TITLE
chore: remove lingering annotations flags

### LINKED ISSUE #2261: Annotations: Remove flags
### Background
Annotations is on in Prod and humming along nicely. It's time to remove the feature flagging code and just have annotations 

### Acceptance Criteria
- [ ] flagging conditions are remove
- [ ] `annotations`, `rangeAnnotations` are removed from flags.yml
- [ ] `annotations`, `rangeAnnotations`, are removed from config cat

### PR DESCRIPTION
**Pull request by hoorayimhelping**
Closes #2261 
Missed some flags with this: https://github.com/influxdata/ui/pull/2306

### CODE REVIEW

File: src/shared/components/graph_tips/GraphTips.tsx [hunk 1]
```diff
@@ -20,21 +19,14 @@ const GraphTips: FC = () => (
...
-          <FeatureFlag name="rangeAnnotations">
-            <div className="graph-tips-line">
-              <code>Shift + Drag</code>{' '}
-              <span> Annotate a range on a graph </span>
-            </div>
-          </FeatureFlag>
+          <br />
+          <span>When Annotations button is on:</span>
+          <div className="graph-tips-line">
+            <code>Shift + Click</code> <span> Annotate a point on a graph</span>
```
Comment (jrenee42) [inline src/shared/components/graph_tips/GraphTips.tsx:25]: why did you remove the spacing? {' '}
(both here and below).  it still looks good/lines up?

**Review by jrenee42 — CHANGES_REQUESTED** [review]

File: src/shared/components/graph_tips/GraphTips.tsx (hunk 1 above)
Comment (hoorayimhelping) [reply src/shared/components/graph_tips/GraphTips.tsx:25]: @jrenee42 prettier did that, not me

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 57 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2316)

**Comment by jrenee42** [comment]
looks good!  just get the tests passing......
